WooCommerce/Classes/ViewRelated/Products/Variations/ProductVariationsViewController.swift

```diff
@@ -269,36 +268,56 @@ private extension ProductVariationsViewController {
     }
 }
 
+// MARK: - Table header view
+//
 private extension ProductVariationsViewController {
     func configureHeaderContainerView() {
         let headerContainer = UIView(frame: CGRect(x: 0, y: 0, width: Int(tableView.frame.width), height: 0))
         headerContainer.addSubview(topStackView)
         headerContainer.pinSubviewToAllEdges(topStackView)
+
+        configureTopButton(title: Localization.addNewVariation,
+                           insets: .init(top: 16, left: 16, bottom: 8, right: 16),
+                           actionSelector: #selector(addButtonTapped),
+                           stylingHandler: { $0.applyPrimaryButtonStyle() })
+
+        configureTopButton(title: Localization.editAttributesAction,
+                           insets: .init(top: 8, left: 16, bottom: 16, right: 16),
+                           hasBottomBorder: true,
+                           actionSelector: #selector(editAttributesTapped),
+                           stylingHandler: { $0.applySecondaryButtonStyle() })
```

Comment:
Nit: Since these functions create separate buttons I think they should be renamed to something like 

```swift
configureTopButton(title: String,
                   insets: UIEdgeInsets,
                   hasBottomBorder: Bool = false,
                   actionSelector: Selector,
                   stylingHandler: (UIButton) -> Void)
```

The current name makes me think that I'm configuring the same top button twice.

What do you think?

Reply:
Good catch! Let me update that.